### PR TITLE
Flashing validation changed to max 120 seconds

### PR DESCRIPTION
**Pull request by VeliMattiLahtela**
Giving flashing all the possible changes to succeed.

### CODE REVIEW

File: mbed_flasher/flashers/FlasherMbed.py [hunk 1]
```diff
@@ -73,18 +73,17 @@ def reset_board(self, serial_port):
...
+            sleep(2)
             if platform.system() == 'Windows':
                 out = os.popen('dir %s' % drive[0]).read()
             else:
                 out = os.popen('ls %s 2> /dev/null' % drive[0]).read()
             if out.find('MBED.HTM') != -1:
                 if out.find(drive[1]) == -1:
                     break
-            if i >= 25:
+            if time() - start_time > 100:
```
Comment (juhhov) [inline mbed_flasher/flashers/FlasherMbed.py]: Avoid hard coding values

File: mbed_flasher/flashers/FlasherMbed.py [hunk 2]
```diff
@@ -254,6 +253,9 @@ def get_me_file():
                                 fault = fault.read().strip()
                             self.logger.error("Flashing failed: %s. tid=%s" % (fault, target))
                             return -4
+                        if isfile(join(mount, tail)):
+                            self.logger.error("Flashing failed: File still present in mount point. tid info: %s" % target)
```
Comment (juhhov) [inline mbed_flasher/flashers/FlasherMbed.py]: Should try to keep rows under 80 characters

File: mbed_flasher/flashers/FlasherMbed.py [hunk 3]
```diff
@@ -14,21 +14,21 @@
...
 import platform
-from time import sleep
+from time import sleep, time
 from threading import Thread
 from enhancedserial import EnhancedSerial
 from serial.serialutil import SerialException
 import hashlib
 import mbed_lstools
 
+FLASHING_VERIFICATION_TIMEOUT = 100
```
Comment (jupe) [inline mbed_flasher/flashers/FlasherMbed.py]: could this be static class member so application can change it..

**Comment by VeliMattiLahtela** [comment]
Unittests added for fail functionality. 
Verifies:
-flash fails if file does not disappear from mount point.
-when flashing fails failure is reported.

**Comment by RomanSaveljev** [comment]
It looks like now tests in raspberry take significantly longer time to run

**Comment by RomanSaveljev** [comment]
Yep, before this PR it takes roughly 140 seconds, but with it over 700. There is 3 more tests, however

**Pull request merged**